engines/sci/sci.cpp

```diff
@@ -436,7 +436,21 @@ Common::Error SciEngine::run() {
 	}
 
 	if (getGameId() == GID_GK2 && ConfMan.getBool("subtitles") && !_resMan->testResource(ResourceId(kResourceTypeSync, 10))) {
-		suggestDownloadGK2SubTitlesPatch();
+		suggestDownloadPatch(_("GK2 has a fan made subtitles, available thanks to the good persons at SierraHelp.\n\n"
+			"Installation:\n"
+			"- download http://www.sierrahelp.com/Files/Patches/GabrielKnight/GK2Subtitles.zip\n"
+			"- extract zip file\n"
+			"- no need to run the .exe file\n"
+			"- extract the .exe file with a file archiver, like 7-zip\n"
+			"- create a PATCHES subdirectory inside your GK2 directory\n"
+			"- copy the content of GK2Subtitles\\SUBPATCH to the PATCHES subdirectory\n"
+			"- replace files with similar names\n"
+			"- restart the game\n"), "http://www.sierrahelp.com/Files/Patches/GabrielKnight/GK2Subtitles.zip");
+	}
+
+	if (getGameId() == GID_GK2 && getGameVersion() == "1.0") {
+		suggestDownloadPatch(_("Your current GK2 version is 1.0 , which is known to suffer from few problems.\n"
```

Comment:
> which is known to suffer from few problems

This makes it sound like version 1.0 only has a few problems, which isn't the case. Either it should be "a few problems" (don't ask me why "few problems" and "a few problems" have different grammatical meaning in English) or rephrase to something like "which has some problems" or "which has known issues".